src/definition/metadata/IAppInfo.ts

```diff
@@ -12,4 +13,5 @@ export interface IAppInfo {
     iconFile: string;
     /** Base64 string of the App's icon. */
     iconFileContent?: string;
+    essentials?: [AppInterface];
```

Comment:
Besides being the "JavaScript" way to do it, any reason why you went with this style instead of `Array<AppInterface>`?

Reply:
This was the first thing that crossed my mind and I haven't considered any other way of doing it until now :P It didn't feel completely right, though. Will switch, thanks